src/test/java/org/mastodon/mamut/collaboration/MastodonGitRepositoryTest.java

```diff
@@ -0,0 +1,314 @@
+package org.mastodon.mamut.collaboration;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashSet;
+
+import org.apache.commons.io.FileUtils;
+import org.eclipse.jgit.api.Git;
+import org.junit.Test;
+import org.mastodon.mamut.ProjectModel;
+import org.mastodon.mamut.collaboration.utils.ModelAsserts;
+import org.mastodon.mamut.collaboration.utils.ModelIO;
+import org.mastodon.mamut.io.ProjectLoader;
+import org.mastodon.mamut.io.ProjectSaver;
+import org.mastodon.mamut.io.project.MamutProject;
+import org.mastodon.mamut.io.project.MamutProjectIO;
+import org.mastodon.mamut.model.Model;
+import org.mastodon.mamut.model.ModelGraph;
+import org.mastodon.mamut.model.Spot;
+import org.scijava.Context;
+
+/**
+ * Tests for {@link MastodonGitRepository}.
+ */
+public class MastodonGitRepositoryTest
```

Comment:
While going through the tests in class and trying to understand what they do, I recognize that many public methods in MastodonGitRepository lack javadoc. While for some methods you can guess from the name, what they do, this is not the case for all of them. Perhaps you could at least create an issue to improve this situation?

Reply:
I added javadoc for all the public methods.